### PR TITLE
{lib}[dummy/] libcpuset v1.0, libbitmask v2.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2539.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8252dbbad5d1d27e3e290f2a5752a810 for a full test report.

File: easybuild/easyconfigs/l/libbitmask/libbitmask-2.0.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+description = "libbitmask provides a convenient, powerful bitmask data type"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['ftp://oss.sgi.com/projects/cpusets/download/']
+sources = [SOURCE_TAR_BZ2]
+
+builddependencies = [('Autotools', '20150215')]
+
+preconfigopts = "libtoolize && aclocal && automake --add-missing && autoconf && "
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libbitmask/libbitmask-2.0.eb]: normally, it's first aclocal, then libtoolize, autoconf and then ony automake?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2130.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/326de44827684754baf5a6be698e3669 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2539.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8add9aeb8cd534f61c0dd3971bd00bfa for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2130.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/dbe6e1359896004019c06bb648be53d2 for a full test report.

**Pull request merged**